doc/api/Track_Selection/setAudioTrack.md

```diff
@@ -116,6 +116,28 @@ rxPlayer.setAudioTrack({
 });
 ```
 
+### Changing the resuming position after an audio track change
+
+In scenarios where a change in media disrupts continuous playback, you can leverage the `relativeResumingPosition` option to set a rewind duration, such as 5 seconds, to revisit the content that might have been cut off during the transition.
+
+The `relativeResumingPosition` is defined as the offset in seconds between the resuming position and the currrent position.
+A negative number will seek back and a positive number will seek forward.
+
+<div class="note">
+The <code>relativeResumingPosition</code> has no effect if:
+
+- The switching mode is set to <code>"seamless"</code>.
+- The track that has been changed is outside of the current period.
```

Comment:
I don't think that stands always true for the worker branch.

I guess we should only say that it will have an effect if playback needs to be interrupted (we can cite seamless and e.g. setting a track initially and adding something like `etc.` or `...` to indicate that there may be more without citing them!)